client/my-sites/domains/domain-management/edit/transfer.jsx

```diff
@@ -48,7 +48,7 @@ class Transfer extends React.PureComponent {
 			noCancelNotice = (
 				<Notice status={ 'is-info' } showDismiss={ false }>
 					{ translate(
-						'This transfer has been started is waiting authorization from your current provider. ' +
+						'This transfer has been started and is waiting for authorization from your current provider. ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **16** times:
`translate( 'This transfer has been started is waiting authorization from your current provider. If you need to cancel the transfer, please contact them for assistance.' )` *ES Score: 16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).